configserver/src/main/java/com/yahoo/vespa/config/server/http/ProxyResponse.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.config.server.http;
+
+import com.yahoo.container.jdisc.HttpResponse;
+import org.apache.http.Header;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Optional;
+
+/**
+ * Proxies response back to client, keeps Content-Type header if it is present
+ *
+ * @author Ola Aunrønning
```

Comment:
Can't really remember, but freva might've been the author of `ProxyResponse`

Reply:
Seems like you both have done work there, I'll add freva as author in next PR